### PR TITLE
Fixed deprecation warnings

### PR DESCRIPTION
**Pull request by gabrielgarciagava**

### CODE REVIEW

**Review by Lyokone — APPROVED** [review]

**Comment by bartekpacia** [comment]
Hi @gabrielgarciagava, would you be up to rebasing this with latest `master`?

File: packages/location/example/macos/Flutter/GeneratedPluginRegistrant.swift
Comment (bartekpacia) [inline packages/location/example/macos/Flutter/GeneratedPluginRegistrant.swift:1]: if you only changed formatting, please revert it

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 This plugin for [Flutter](https://flutter.io)
 handles getting a location on Android and iOS. It also provides callbacks when the location is changed.
 
+**[Version 5](https://github.com/Lyokone/flutterlocation/pull/730) of the plugin is under construction, so new features will not be merged in version 4 for the time being.**
```
Comment (bartekpacia) [inline README.md:9]: remove this as it's no longer true

**Comment by bartekpacia** [comment]
no response

**Pull request closed without merging**